test/GovernanceStaker.t.sol

```diff
@@ -2764,6 +2764,58 @@ contract NotifyRewardAmount is GovernanceStakerRewardsTest {
     assertLe(govStaker.scaledRewardRate(), _expectedRewardRate);
   }
 
+  function test_shareManipulationIfRewardsArentCeckpointed() public {
+    earningPowerCalculator.__setEarningPowerForDelegatee(address(0x1), 0);
+    earningPowerCalculator.__setEarningPowerForDelegatee(address(0x2), 500e18);
+
+    address _doe = makeAddr("doe");
+    _mintGovToken(_doe, 500e18);
+    _stake(_doe, 500e18, address(0x2));
+
+    address _fox = makeAddr("fox");
+
+    //!audit-ok Step 1, fox deposits with full earning power
```

Comment:
might revise these comments, or possibly attribute [CergyK](https://github.com/CergyK) above the test

Reply:
Updated